React/CSS-in-JS/Chakra.md

```diff
@@ -251,6 +327,59 @@ Array and Object syntax work for every style prop in the theme specification, wh
 
 To create custom breakpoints read [docs](https://chakra-ui.com/docs/features/responsive-styles#customizing-breakpoints).
 
+For more about responsive styles read [styled-system responsive styles docs](https://styled-system.com/responsive-styles/)
+
+### Chakra Factory
+
+Chakra factory serves as an object of chakra JSX elements, and also a function that can be used to enable custom component receive chakra's style props.
+
+```tsx
+import { chakra } from "@chakra-ui/react";
+
+// object
+<chakra.button
+  sx={{
+    shadow: "lg",
+    rounded: "lg",
+    bg: "white",
+  }}
+>
+  Click me
+</chakra.button>;
+
+// function
+chakra("button", {
+  baseStyle: {
+    shadow: "lg",
+    rounded: "lg",
+    bg: "white",
+  },
+});
+```
+
+This reduces the need to create custom component wrappers and name them. Syntax is available for common html elements. See the reference for the full [list of elements](https://github.com/chakra-ui/chakra-ui/blob/develop/packages/system/src/system.utils.ts#L9) supported.
+
+Chakra factory function can also that convert jsx element to chakra-enabled components so you can pass style props to them.
```

Comment:
This is not clear enough. I don't understand what you want to say

Reply:
I removed this because we have something similar few lines above.